modules/opus/encode.c

```diff
@@ -100,8 +100,7 @@ int opus_encode_update(struct auenc_state **aesp, const struct aucodec *ac,
 		aes->ch = ac->ch;
 
 		aes->enc = opus_encoder_create(ac->srate, ac->ch,
-					       /* this has big impact on cpu */
-					       OPUS_APPLICATION_AUDIO,
+					       OPUS_APPLICATION_VOIP,
```

Comment:
The value should then also be configurable imho. I prefer better quality over cpu usage in my project.

Reply:
I don't mind creating such a config param.

Reply:
> The value should then also be configurable imho. I prefer better quality over cpu usage in my project.

I've added such parameter which defaults to `OPUS_APPLICATION_AUDIO`